src/animation/AnimationUtils.js

```diff
@@ -231,6 +233,105 @@ var AnimationUtils = {
 
 		return clip;
 
+	},
+
+	makeClipAdditive: function ( sourceClip, referenceFrame = 0, cloneOriginal = false, clonedName, fps = 30 ) {
```

Comment:
I would suggest omitting the 'cloneOriginal' and 'clonedName' parameters. The user can clone or rename the source clip if they would like to do so.